### PR TITLE
Feature/simplify card

### PR DESCRIPTION
**Pull request by zacoppotamus**
Publishes v.0.6.0. Deprecates hacky approaches for rendering Card fields, in favor of a DSL-like approach that controls the layout. Transitioning to the new API for <Card /> breaks the other stories, along with inline media, so these will need to be updated.

### CODE REVIEW

**Comment by breezykermo** [comment]
Much simpler! How feasible would it be to reinstate the stories with inline media and sources before merge?

**Comment by zacoppotamus** [comment]
Will do!

File: src/lib/components/Card/Card.stories.js [hunk 1]
```diff
@@ -5,66 +5,42 @@ import { CardProps } from "../../../test/Card";
...
 TextOnly.args = {
   ...CardProps,
 };
 
-export const InlineVideoPortrait = Template.bind({});
-InlineVideoPortrait.args = {
-  ...CardProps,
-  renderOrder: [[`renderContextual`], [`renderMedia`], [`renderSummary`]],
-  renderExtra: [],
+console.log(CardProps);
```
Comment (breezykermo) [inline src/lib/components/Card/Card.stories.js:13]: hanging console.log here

File: src/lib/components/Card/Card.stories.js [hunk 2]
```diff
@@ -5,66 +5,42 @@ import { CardProps } from "../../../test/Card";
...
-    ...changeMediaSrc(
-      CardProps.event,
-      `https://datasheet-sources.ams3.cdn.digitaloceanspaces.com/us2020/114.mp4`
-    ),
-  },
+export const InlineVideoPortrait = Template.bind({});
+InlineVideoPortrait.args = {
+  ...addInlineMedia(
+    CardProps,
+    "https://datasheet-sources.ams3.cdn.digitaloceanspaces.com/us2020/114.mp4"
```
Comment (breezykermo) [inline src/lib/components/Card/Card.stories.js:26]: Is this hardcoded link a default URL? If so, perhaps we host two or three images associated with the ‘timemap_data’ example sheet, and put this here.

File: src/lib/components/Card/Card.stories.js [hunk 3]
```diff
@@ -5,66 +5,42 @@ import { CardProps } from "../../../test/Card";
...
-  renderExtra: [],
-  event: {
-    ...changeMediaSrc(
-      CardProps.event,
-      `https://datasheet-sources.ams3.cdn.digitaloceanspaces.com/us2020/118.jpg`
-    ),
-  },
+  ...addInlineMedia(
+    CardProps,
+    "https://datasheet-sources.ams3.cdn.digitaloceanspaces.com/us2020/118.jpg"
```
Comment (breezykermo) [inline src/lib/components/Card/Card.stories.js:34]: Likewise to the comment on videos above. Good for now, though.

**Review by breezykermo — APPROVED** [review]
Ok nice, this looks like some good cleaning. So we’re sticking with the DSL, and timemap will generate this from some sort of slimmer config before passing as props to the Card here in the design system?

**Comment by breezykermo** [comment]
And just to confirm- will there also be a PR to timemap that removes the hardcoded `us2020` logic, refactoring it to the config in some way, and adding some more config-agnostic code for generating the appropriate DSL to pass here to the `<Card/>`?

**Pull request merged**